terasoluna-gfw-functionaltest-selenium/src/test/java/org/terasoluna/gfw/functionaltest/app/FunctionTestSupport.java

```diff
@@ -187,14 +187,15 @@ protected void bootDefaultWebDriver() {
         if (driver == null) {
             driver = newWebDriver();
         }
-        driver.manage().timeouts().implicitlyWait(
-                defaultTimeoutSecForImplicitlyWait, TimeUnit.SECONDS);
+        driver.manage().timeouts().implicitlyWait(Duration.ofSeconds(
+                defaultTimeoutSecForImplicitlyWait));
         driver.get(getPackageRootUrl());
 
         this.webDriverOperations = new WebDriverOperations(driver);
         this.webDriverOperations.setDefaultTimeoutForImplicitlyWait(
                 defaultTimeoutSecForImplicitlyWait);
-        this.webDriverWait = new WebDriverWait(driver, defaultTimeoutSecForImplicitlyWait);
+        this.webDriverWait = new WebDriverWait(driver, Duration.ofSeconds(
+                defaultTimeoutSecForImplicitlyWait));
```

Comment:
`Duration.ofSeconds(defaultTimeoutSecForImplicitlyWait)` is used multiple times, so it is better to define a variable.

Reply:
`Duration.ofSeconds` internal implementation would create instances needlessly, so let's cut it out into a variable.

Reply:
Fixed with a20f7a7c18f13432b4cbb129a57a352704932c3b in #986